pyup/providers/gitlab.py

```diff
@@ -184,7 +184,7 @@ def create_pull_request(self, repo, title, body, base_branch, new_branch, pr_lab
                 'title': title,
                 'description': body,
                 'pr_label': pr_label,
-                'should_remove_source_branch': config.gitlab.should_remove_source_branch
+                'remove_source_branch': config.gitlab.remove_source_branch
```

Comment:
I suggest to reduce this change to only this line - which is the actual fix.

Reply:
Hi. Are you `pyup` commiter? The repository seems kind of dead.

Reply:
@WouldYouKindly: Nope, I'm just a user

Reply:
That sounds like the best option. And sorry for the late response, @WouldYouKindly